src/facade/memcache_parser_test.cc

```diff
@@ -146,4 +146,6 @@ TEST_F(MCParserNoreplyTest, LargeGetRequest) {
   }));
 }
 
+TEST_F(MCParserTest, SetLength) {
```

Comment:
indeed it's part of dragonfly_connection.cc right  so we should write a test in pytest using raw socket interface. the protocol is easy, so the test should be easy to write.

Reply:
https://chat.openai.com/share/1cbf6e42-6800-4ead-9e07-171c145d3b5b here is how